specs/binding-grpc.spec/src/main/scripts/io/aklivity/zilla/specs/binding/grpc/schema/grpc.schema.patch.json

```diff
@@ -124,6 +125,35 @@
                                     }
                                 },
                                 "additionalProperties": false
+                            },
+                            "catalog":
```

Comment:
This is the same structure and schema that is found in the `asyncapi` and `openapi` bindings and matches the `GrpcCatalogSchema` class